src/Rules/Library/ListItemSiblingsUnique.cs

```diff
@@ -32,9 +34,9 @@ public override EvaluationCode Evaluate(IA11yElement e)
                 & Name.Is(e.Name)
                 & LocalizedControlType.Is(e.LocalizedControlType));
             var count = siblings.GetValue(e);
-            if (count < 1) return EvaluationCode.RuleExecutionError;
+            if (count < 1) throw new Exception(String.Format(CultureInfo.CurrentCulture, ErrorMessages.NoElementFound, this.Info.ID));
```

Comment:
Not entirely sure why the Exception is different, but I'd suggest CultureInfo.InvariantCulture instead of CultureInfo.CurrentCulture (the message isn't localized, so it doesn't make sense to localize the ID)

Reply:
Why the exception is different from what?

And I am going to fix these instances in a single change so I can make sure they compile.

Reply:
I understand the Exception now--you're throwing here and RuleExecutionError will come from the Exception handler.